src/common/snippets/src/pass/mha_tokenization.cpp

```diff
@@ -358,6 +363,7 @@ ov::snippets::pass::TokenizeMHASnippets::TokenizeMHASnippets(const SnippetsToken
         // There is transformation ExplicitTransposeMatMulInputs that set supported order and transposed_b(false).
         // We can allow to call this pass only if ops have scalar shapes to avoid shape mismatching
         const auto is_transposed_b_0 = matmul0->get_transpose_b();
+        bool has_matmul0_has_ops_on_input = false;
```

Comment:
Minor typo
```suggestion
        bool matmul0_has_ops_on_input = false;
```

Reply:
If CI is red, I will fix in this PR. Otherwise I'd suggest to fix in the follow-up PR 😄

Reply:
Of course)